### DISCUSSION TITLE
ND and PFD not doing a self-test

### DISCUSSION THREAD

**Discussion by yukikaze04** [flybywiresim/aircraft#2326] [Help]
Just started cold and dark, went for the external power, ND and PFD on both sides did not do a self-test, other screens did.
When I powered down, and turned external power on again, self-test happened on all screens correctly. I don't remember this happening before, just now with the new asobo hotfix and latest dev build at the time of writing.

**Comment by Benjozork** [comment]
Do you have any other mods installed ?
